mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -272,15 +285,14 @@ class Word2Vec extends Serializable with Logging {
 
   /**
    * Computes the vector representation of each word in vocabulary.
-   * @param dataset an RDD of words
+   * @param dataset a RDD of sentences,
```

Comment:
This is an interesting topic, seem r is not a vowel, not sounds like vowel either, why 'an'?
I found this from web:You use the article “a” before words that start with a consonant sound and “an” before words that start with a vowel sound.